bluesky/suspenders.py

```diff
@@ -148,6 +158,11 @@ def __set_event(self):
                 loop = self.RE._loop
 
                 def local():
+                    ts = ((datetime.now() + timedelta(seconds=sleep))
+                           .strftime('%Y %m %d %H:%M:%S'))
+                    print("Suspender {!r} reports a return to nominal "
+                          "conditions. Will sleep for {!r} seconds and then "
+                          "release suspension at {!r}".format(self, ts))
```

Comment:
I caught this on the train but didn't have time to push. Sorry for not speaking up, and thanks for looking into it anyway.